client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/style.scss

```diff
@@ -13,7 +13,80 @@ button.configure-contact__button {
 	padding: 4px 8px !important;
 }
 
-.configure-contact__form {
+$help-text-color: #757575;
+
+.configure-contact-info__card {
+	margin: 0;
+	height: 60px;
+	display: flex;
+	align-items: center;
+	padding: 16px !important;
```

Comment:
Main comment to address a few examples in this same file:

Let's avoid the use of `!important` if at all possible -- it makes overriding these rules in the future very confusing/difficult.

An alternate approach would be to add more specificity to this rule set, like specifying another class, ID, or element. In a pinch, when there's no other identifier to apply, you can even double up on the same one (e.g., `.configure-contact-info__card.configure-content-info__card`).